website/docs/reference/resource-configs/updated_at.md

```diff
@@ -27,6 +27,12 @@ snapshots:
 
 </File>
 
+:::caution
+
+You might get a warning if the data type of the `updated_at` column does not match the adapter-configured default.
```

Comment:
@dbeatty10 Hi! This is for [this warning](https://github.com/dbt-labs/dbt-core/issues/10234) - what do you think about this wording?

Reply:
Is there any scenario in which they _wouldn't_ get a warning about the mismatch?

If not, then I'd suggest updating "might" to be "will" instead.

Other than that, this call-out looks good! 👍